src/src-admin/views/Logout.js

```diff
@@ -0,0 +1,58 @@
+/* global google */
```

Comment:
Logout stuff again with lot's of unused/ commented out functionality. Not sure if going to be used in the future?